src/utils/SchemaReference.js

```diff
@@ -89,7 +90,7 @@ export function getArgumentReference(typeInfo: any): ArgumentReference {
       };
 }
 
-export function getEnumValueReference(typeInfo: any): EnumValueReference {
+export function getEnumValueReference(typeInfo: any): any {
```

Comment:
Maybe a $FlowFixMe comment would be more appropriate? (What was the error?)

Reply:
If you run `yarn check` on the master branch, youll see about 24 flow errors total!

But $FlowFixMe sounds good for now 👍

Reply:
the error is with `getNamedType`, where because `typeInfo` is `any`, it infers this definition from `graphql-js`: 
`declare function getNamedType(type: void | null): void;`
instead of:
`declare function getNamedType(type: GraphQLType): GraphQLNamedType;`
and `EnumValueReference` wants a `type` value of `GraphQLEnumType`.

the proper, long term solution IMO would be to specify types for `typeInfo` across this project, but thats a big undertaking

Reply:
Maybe re-cast it internally?

```suggestion
export function getEnumValueReference(typeInfo: any): any {
  const graphqlEnumType: GraphQLEnumType = typeInfo.inputType;
```

Reply:
hmmm... thats a good idea

Reply:
![Screen Shot 2019-05-28 at 10 27 11 AM](https://user-images.githubusercontent.com/1368727/58486069-2ffdfb00-8133-11e9-90d3-37ecb1ef3087.png)

Reply:
Possibly the issue here is using `getNamedType`?

Reply:
I'll try and take a look tomorrow.